src/utils/Processing.ts

```diff
@@ -26,13 +26,12 @@ export function getResources(inDir: string, defs: fhirdefs.FHIRDefinitions) {
   logger.info(`Found ${files.length} JSON files.`);
   files.forEach(file => {
     try {
-      processor.process(file, resources.invariants).forEach(resource => {
-        resources.add(resource);
-      });
+      processor.register(file);
     } catch (ex) {
-      logger.error(`Could not process ${file}: ${ex.message}`);
+      logger.error(`Could not register ${file}: ${ex.message}`);
```

Comment:
I'm not sure if the word "register" will mean much to our users.  Maybe "Could not _load_ foo.json"?